### PR TITLE
Message removing timer

### PR DESCRIPTION
**Pull request by 42-alex**

### CODE REVIEW

File: front/src/hooks.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import { useDispatch } from 'react-redux';
+import { AppDispatch } from './redux/store';
+
+export const useAppDispatch: () => AppDispatch = useDispatch;
```
Comment (skyboyer) [inline front/src/hooks.ts]: what is the reason/need behind this wrapper?

**Review by skyboyer — APPROVED** [review]
looks good but I did not understand the reasoning behind useAppDispatch

File: front/src/hooks.ts (hunk 1 above)
Comment (42-alex) [reply front/src/hooks.ts]: @skyboyer 
It creates the typed dispatch() for you and allows you dispatch a thunk ([link](https://redux-toolkit.js.org/usage/usage-with-typescript#getting-the-dispatch-type ))

Comment (skyboyer) [reply front/src/hooks.ts]: ah, I see. all is good then

**Pull request merged**